content/en/how-to-guides/configuration-management/change_management.adoc

```diff
@@ -0,0 +1,60 @@
+
+  * Source code related issues should be created in the https://github.com/tetrabiodistributed/project-tetra/issues[Project Tetra issues] and added to the https://github.com/orgs/tetrabiodistributed/projects/1[Project Tetra - kanban board].
+
+  * Non-documentation or source code related issues should be added as a card to the https://github.com/orgs/tetrabiodistributed/projects/1[Project Tetra - kanban board].
+
+. Record relevant discussions in the comment section of the GitHub issues.
+
+== How to make documentation changes ==
+
+. Create a new branch, or checkout an existing branch, in the https://github.com/helpfulengineering/project-tetra-docs[Project Tetra docs repo]. The branch name should describe the changes to be made, followed by its associated issue number, e.g., `add-tricoder-with-touchscreen-docs-1701`. If no such issue exists, <<_creating_an_issue, create a new issue>>.
+
+. Write or update the appropriate documentation page relevant to the design changes to be made in your working branch.  All documentation should be nested under link:../../tutorials[Tutorials], link:../../how-to-guides[How-To Guides], link:../../explanations[Explanations], and/or link:../../references[References]. (See <<#_how_to_write_good_documentation, How to write good documentation>>)
+
+. Commit the documentation changes to your working branch in the https://github.com/tetrabiodistributed/project-tetra-docs[Project Tetra docs repo].
+
+. Create a pull request from your working branches in the https://github.com/tetrabiodistributed/project-tetra-docs[Project Tetra docs repo] and https://github.com/helpfulengineering/project-tetra[Project Tetra repo] to be merged into their respective main branch.
+
+== How to make design changes ==
+
+Design changes include changes to CAD, STL, Drawings, or software.
+
+. Update the documentation to reflect the design changes to be made. (See <<_how_to_make_documentation_changes, How to make documentation changes>>)
+
+. Create a new working branch, or checkout an existing branch, in the https://github.com/helpfulengineering/project-tetra[Project Tetra repo]. The branch name should describe the change to be made, followed by its associated issue number, e.g., `add-tricoder-with-touchscreen-1031`. If no such issue exists, create a new issue in the https://github.com/tetrabiodistributed/project-tetra/issues[Project Tetra issues].
++
+Note: Part number revisions and version releases should accompany a design change where appropriate. (See link:../version_control/[Version Control])
+
+. Commit your changes and create a pull request (PR).
+
+. If you're not yet ready for a review, mark the PR as _Draft_ to indicate it's a work in progress. Continue updating your doc and pushing your changes until you're happy with the content.
```

Comment:
Hardware reviews will happen before PR

Reply:
But the whole point of opening a PR is to indicate that we're requesting a (hardware) review. If the reviewer says there needs to be changes, the PR author needs to keep implementing those changes until the reviewer is satisfied.

Reply:
On second thought, I can see why you'd want to do hardware reviews before opening a PR, because opening a PR would mean having to commit files to a branch at the possibility that they need to be re-added & committed in the event there are changes that need to be made. So yeah, I retract my original comment.